Week2/Bank.java

```diff
@@ -0,0 +1,118 @@
+package Week2;
+
+import java.util.*;
+import java.math.BigDecimal;
+//  계좌생성, 출금, 입금, 송금 메서드 구현
+// 내부에서 입력을 받는 액션?
+// 적금 계좌는
+public class Bank {
+    private ArrayList<Account> accountArray;

```

Comment:
```suggestion
    private final ArrayList<Account> accountArray = new ArrayList<>();

```
가독성과 코드 중복 방지를 위해 필드에서 초기화 하는 것이 좋습니다.
예를 들어 여러 개의 생성자를 만들 경우, 각 생성자 마다 초기화 해야하는 코드 중복이 발생합니다
final 키워드를 사용해 변경되지 않을 인스턴스라는 것을 명시해야 합니다.